e2e/scenarios/raceCondition.spec.ts

```diff
@@ -0,0 +1,242 @@
+import { test } from "@playwright/test";
+import {
+  addAndRemoveTrack,
+  addAndReplaceTrack,
+  addBothMockTracks,
+  assertThatAllTracksAreReady,
+  assertThatBothTrackAreDifferent,
+  assertThatOtherVideoIsPlaying,
+  assertThatTrackBackgroundColorIsOk,
+  assertThatTrackReplaceStatusIsSuccess,
+  clickButton,
+  createAndJoinPeer,
+  createRoom,
+  removeTrack,
+  takeScreenshot,
+} from "./utils";
+
+test.afterEach(async ({ context }, testInfo) => {
+  for (const page of context.pages()) {
+    await test.step("Screenshot after test", async () => {
+      await takeScreenshot(page, testInfo, "After test Screenshot");
+    });
+  }
+});
+
+/*
+ * Test in this file should be run a few times in a row to be sure that there is no race conditions.
+ * To run a test multiple times use command:
+ *
+ * npm run e2e -- --repeat-each=20 -g "Test name"
```

Comment:
https://playwright.dev/docs/test-parameterize#parameterized-tests Couldn't we just express it with code?

Reply:
For convenience, I've added `e2e:repeat` to package json with this command: `playwright test --repeat-each=20`